cli/src/cluster_query.rs

```diff
@@ -102,6 +102,24 @@ impl ClusterQuerySubCommands for App<'_, '_> {
                         .index(1),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("recent-prioritization-fees")
+                .about("Get recent prioritization fees")
+                .arg(
+                    Arg::with_name("accounts")
+                        .long("accounts")
+                        .value_name("ACCOUNTS")
+                        .takes_value(true)
+                        .multiple(true)
+                        .index(1),
+                )
+                .arg(
+                    Arg::with_name("num_slots")
```

Comment:
this name kinda implies that one can specify more, when in fact we can only limit the displayed slots. `--limit-n-slots`?

Reply:
changed to that